README.md

```diff
@@ -26,3 +26,21 @@ Debian build facilities are integrated into the repo at `debian/`.
 3. Install the resulting `jellyfin*.deb` file on your system.
 
 A huge thanks to Carlos Hernandez who created the Debian build configuration for Emby 3.1.1.
+
+### Windows (64 bit)
+A pre-built windows installer will be available at [The JellyFin Repository](https://repo.jellyfin.org/).
```

Comment:
It may be worth emphasising that this currently isn't available, and if possible adding a timeline as to when it will be.